versions/models.py

```diff
@@ -557,8 +584,8 @@ def create_versioned_many_to_many_intermediary_model(field, cls, field_name):
         return type(str(name), (Versionable,), {
             'Meta': meta,
             '__module__': cls.__module__,
-            from_: VersionedForeignKey(cls, related_name='%s+' % name),
-            to_field_name: VersionedForeignKey(to, related_name='%s+' % name),
+            from_: VersionedForeignKey(cls, related_name='%s+' % name, auto_created=name),
```

Comment:
Is this abusing the auto_created property?  I'm not really familiar with auto_created.  I'm just concerned if it will cause problems in some situations if it's not being used as Django expects; what comes to mind is migrations.